### PR TITLE
docs: Added notebook guidelines in README

### PR DESCRIPTION
**Pull request by fg-mindee**
As per @felixdittrich92 suggestion, this PR adds some notebook guidelines to the README.

Any feedback is welcome!

### CODE REVIEW

**Comment by felixdittrich92** [comment]
@fg-mindee 
Thanks 👍

**Review by fharper — DISMISSED** [review]
LGTM, add @fakela as a reviewer before you are ready to merge so she can proofread it :)

**Comment by fg-mindee** [comment]
ping @charlesmindee & @SiddhantBahuguna for review also :pray:

**Comment by fg-mindee** [comment]
@fakela @charlesmindee @SiddhantBahuguna would you mind reviewing it please? :pray: 
(if you don't have time, I can merge it otherwise)

File: README.md
Comment (fakela) [inline README.md]: changing **as is** -> **arrangement:** _please move all imports up into a single cell, and ensure that the notebook can be run successfully with the cell order arrangement._

File: README.md
Comment (fakela) [inline README.md]: changed  **comma** position: _your content will help other users who will have different coding skills, so please don't refrain from adding explanations in text cells (hyperlinks if necessary, etc.)._

File: README.md
Comment (fakela) [inline README.md]: watch out for your grammar; for example, sentences should begin with an uppercase letter and end with a dot, etc.

File: README.md
Comment (fakela) [inline README.md]: the golden rule in open source; remember to credit all external works that you've been using.

**Comment by SiddhantBahuguna** [comment]
> @fakela @charlesmindee @SiddhantBahuguna would you mind reviewing it please? pray (if you don't have time, I can merge it otherwise)

Yes, I am on it now.

File: README.md
Comment (SiddhantBahuguna) [inline README.md:34]: Something about the choices available to upload the sample image for demonstration purposes.
Our preferred way or at least something common to all the notebooks till now is to download the image at the time of runtime from publicly available sites. Either that, or may be we should upload it on drive and share it to public.

File: README.md
Comment (SiddhantBahuguna) [inline README.md:24]: Along with a good title, perhaps, it might be better to include a short introduction on the topic and general outline of the description of the model used with proper references.

File: README.md
Comment (fg-mindee) [reply README.md:34]: Oh true :+1:

File: README.md
Comment (fg-mindee) [reply README.md:24]: Good idea :+1:

**Comment by fharper** [comment]
I'll wait for you to resolved actual conversation before reviewing again.

**Review by SiddhantBahuguna — APPROVED** [review]
Looks Good! Thanks:)

**Comment by fharper** [comment]
@fg-mindee: can we merge this one?

**Comment by fg-mindee** [comment]
@fharper sure, but the PR requires an approval from either you @fakela or @charlesmindee to be merged :sweat_smile:

**Review by fakela — APPROVED** [review]

**Pull request merged**
